src/anndata/_io/specs/lazy_methods.py

```diff
@@ -67,13 +67,18 @@ def make_dask_chunk(
     *,
     wrap: Callable[[ArrayStorageType], ArrayStorageType]
     | Callable[[H5Group | ZarrGroup], _CSRDataset | _CSCDataset] = lambda g: g,
+    reopen: None | Callable[[], Iterator[StorageType]] = None,
```

Comment:
so the idea is “reopen is a callable that can be transformed into a context manager using `contextlib.contextmanager`.

Why not just “reopen is a callable that returns an [`contextlib.AbstractContextManager[StorageType]`](https://docs.python.org/3/library/contextlib.html#contextlib.AbstractContextManager)?